content/500-platform/550-platfrom-cli/commands.mdx

```diff
@@ -0,0 +1,230 @@
+---
+title: 'Commands'
+metaTitle: 'Platform CLI: Commands'
+earlyaccess: true
+tocDepth: 5
```

Comment:
do we need this at 5? I think we probably don't want to show the **Arguments** section for each command in the TOC, so I guess 4 should be enough then?